package.json

```diff
@@ -52,6 +52,8 @@
     "test:ui": "grunt test:ui",
     "test:ui:server": "grunt test:ui:server",
     "test:ui:runner": "echo 'use `node scripts/functional_test_runner`' && false",
+    "test:ui:wdio": "grunt test:ui:wdio",
+    "wdioTests": "node scripts/wdio",
```

Comment:
What's the role of each of these scripts?

Reply:
So, `wdioTests` is called by `grunt test:ui:wdio` after starting all of the servers and stuff.

Reply:
Probably a better way to orchestrate that.